deals/service.go

```diff
@@ -4,7 +4,6 @@ import (
 	"context"
```

Comment:
All this changes are adjustments to a change in the `deals.Store(..)` api that merged two output lists into a single one (as mentioned in an earlier comment).